src/proxy/ClientBuild.php

```diff
@@ -59,6 +65,10 @@ public function __construct(Command $command, array $config = [])
     {
         $this->command = $command;
         parent::__construct($config);
+
+        if (!$this->db) {
+            $this->db = Yii::$app->db;
```

Comment:
For the client side (the command) we should use the --db=xyz option as discussed. So this way we all the options: